roles/pbx/tasks/main.yml

```diff
@@ -21,6 +21,11 @@
 
 - block:
 
+  - name: EXIT 'pbx' ROLE, if 'php_version' doesn't shows "7.4"
+    fail:    # FORCE IT RED THIS ONCE!
+      msg: FreePBX 16 requirs PHP 7.4 and is not available.
+    when: php_version is version('8.0', '>') and pbx_php74_override is undefined
```

Comment:
```suggestion
    when: php_version is version('8.0', '>=') and pbx_php74_override is undefined
```

Reply:
Which OS that we support is shipping version 8.0?

Reply:
This would of been easy just to check OS.yml but that went by the wayside

Reply:
The 2 explanations (`name:` and `msg:`) should match what the actual `when:` condition does.

(PHP 7 versus 8 is just one very practical threshold to possibly consider, to avoid getting lost in the reeds/weeds.)

Reply:
Would you prefer 31a1664483bba99ff6250ef19c2afbf698248599 ?

Reply:
Conditioning on PHP 7.4 is much more granular than necessary, but that can do the job.

Reply:
I'll leave that for you to alter after this PR is pushed like you usually do anyway. Not going to waste my time debating that the sky is green... Sure it's green. Whatever...